lib/web_ui/lib/src/engine/text_editing.dart

```diff
@@ -61,10 +61,20 @@ class EditingState {
   ///   "composingExtent": -1
   /// }
   /// ```
-  EditingState.fromFlutter(Map<String, dynamic> flutterEditingState)
-      : text = flutterEditingState['text'],
-        baseOffset = flutterEditingState['selectionBase'],
-        extentOffset = flutterEditingState['selectionExtent'];
+  ///
+  /// Flutter Framework can send the selectionBase and selectionExtent as -1,
+  /// if so 0 assigned to baseOffset and extentOffset. -1 is not a valid
```

Comment:
nit: if you don't want to make identifier reference linkable, I'd recommend using plain English descriptors in docs, e.g. "base offset" instead of "baseOffset", and "extent offset" instead of "extentOffset".

Reply:
Ah, I forgot the [ ] . Thanks!